Loader/SymfonyFixturesLoader.php

```diff
@@ -28,14 +29,17 @@
      */
     public function addFixtures(array $fixtures)
     {
-        // Store all loaded fixtures so that we can resolve the dependencies correctly.
-        foreach ($fixtures as $fixture) {
-            $class = get_class($fixture['fixture']);
-            $this->loadedFixtures[$class] = $fixture['fixture'];
-            $this->addGroupsFixtureMapping($class, $fixture['groups']);
+        // Because parent::addFixture may call $this->createFixture
+        // we cannot call $this->addFixture in this loop
+        foreach ($fixtures as $fixtureData) {
+            $fixture = $fixtureData['fixture'];
+            $class = get_class($fixture);
+            $this->addGroupsFixtureMapping($class, $fixtureData['groups']);
+            $this->loadedFixtures[$class] = $fixture;
```

Comment:
Did you change anything here? Or just reorder and rename things? If you just renamed things, could we change it back? It’s hard to focus on the important things that you added. But if you did make a significant change here, could you explain it?

Thanks :)

Reply:
Yes, you are right. It was over a couple of commits. (If you follow my commits you will see.) But yes I ended up just reordering and renaming that part. Sorry about that, was not my intention.